src/handwritten/task.py

```diff
@@ -1007,6 +1007,8 @@ def write(self, data, auto_start=AUTO_START_UNSET, timeout=10.0):
 
         # Analog Input
```

Comment:
This comment is wrong, and also not useful. The if statement is self-documenting. Just delete it and any other similar ones in this function.

Reply:
Removed unnecessary comments.